### ISSUE TITLE
Map of the Green Software Ecosystem – Organizations

### ISSUE THREAD

**Issue by holanita** [Green-Software-Foundation/state-of-green-software#78]
Hey @camcash17 feel free to add your findings to this issue and we'll incorporate this into our agenda.

**Comment by holanita** [comment]
Hey all, as suggested by Chris in the call today, let's continue the discussion here async. Let me share the content I have. 
- @mrchrisadams is this the old document you mentioned in the call? [these](https://docs.google.com/document/d/1fLespnMFXw4CjVqmcZ9ktxRsIw76ierQhNliERFQAXw/edit?disco=AAAAlSI9SUk) might be the categories you mentioned
- @camcash17 thanks a lot for the document you've put together so far. [here is the link](https://docs.google.com/document/d/10dkjjfQqnwvriUc7XKHkbjoPNsinbrNQhjljotXnsiU/edit) for the team to review
- @Leamsi9 [this is the link](https://docs.google.com/document/d/1AhkCZWPcdFJBV_nk27xhip-2aiW-RBmb/edit#bookmark=id.hcwuudvztal3) to where you will be fleshing out your content. please update when you can.

@mrchrisadams let us know re: the categories so we can proceed in line with your expectations. cc @jawache

**Comment by jawache** [comment]
Thanks @holanita and @mrchrisadams - my primary concern is that we actually **do** create a landscape analysis as part of the SoGS report which meets the original requirements. The intention was for that report to become a sub-report of the SoGS report. So would be good if @mrchrisadams you could review this part of the report as early as possible and course correct if it's going in a wrong direction? 

Also categorizing is useful, but just listing them adds a ton of value all by itself. There is not even a list of organizations that are interested in green software.

**Comment by holanita** [comment]
@jawache just to make sure we are on the same page, is [this](https://docs.google.com/document/d/1AhkCZWPcdFJBV_nk27xhip-2aiW-RBmb/edit#bookmark=id.hcwuudvztal3) what you expect @mrchrisadams to review?

**Comment by jawache** [comment]
@holanita yes but specifically in that list the original "landscape analysis" = "Map of stakeholders" so that specific sub-section of the whole Map of the Green Software Ecosystem

**Comment by holanita** [comment]
Hi all – just had a catch up with @jawache about this. @camcash17 would you be able to continue working on the stakeholder map in line with the grouping that @mrchrisadams mentioned above? Let us know if you have any questions or need any clarifications!

**Comment by ccasher** [comment]
decided to shift over to a [spreadsheet](https://docs.google.com/spreadsheets/d/1x4bM626wGRRFneThzcqQIv2GQNW5vIdEtnpjFfbjNo0/edit#gid=0) format to be able to add rows and columns pertaining to groupings. Would appreciate some feedback on this approach!

**Comment by jawache** [comment]
Works for me @camcash17. 

In terms of sourcing other names what do you think? We should ask internally at least.

There is also now the environmental TAG in cncf.

And also FINOS/OPS? has an environmental WG.

Obviously green web foundation.

Carbon trust is doing some work with scope 3 energy usage for applications.

Green coding in Berlin I think is a meetup community

Boavista from France.

SDIA.

Sorry just trying to make some notes while I'm looking after the kids little one!
